src/main/resources/default/templates/biz/model/query-results.html.pasta

```diff
@@ -4,61 +4,97 @@
-        </div>
-    </i:for>
+                                            <i:if test="property.is(sirius.db.mixing.properties.NestedListProperty.class)">
+
+                                                <i:invoke
+                                                        template="/templates/biz/model/query-nested-property.html.pasta"
+                                                        property="property.as(sirius.db.mixing.properties.NestedListProperty.class)"
+                                                        entity="@entity"/>
+
+                                                <i:else>
+                                                    <i:if test="property.is(sirius.db.mixing.properties.BaseMapProperty.class)">
+
+                                                        <i:invoke
+                                                                template="/templates/biz/model/query-map-property.html.pasta"
+                                                                property="property.as(sirius.db.mixing.properties.BaseMapProperty.class)"
+                                                                entity="@entity"/>
 
-</w:page>
+                                                        <i:else>
+                                                            <i:if test="property.is(sirius.db.mixing.properties.BaseEntityRefListProperty.class)">
+                                                                <i:invoke
+                                                                        template="/templates/biz/model/query-ref-list-property.html.pasta"
+                                                                        property="property.as(sirius.db.mixing.properties.BaseEntityRefListProperty.class)"
+                                                                        entity="@entity"/>
 
-<script type="text/javascript">
-    $(document).ready(function () {
-        $('#editForm').keydown(function (e) {
-            if (e.keyCode === sirius.keys.KEY_ENTER) {
-                $('#editForm').submit();
-                return false;
-            }
-        });
-    });
-</script>
+                                                                <i:else>
+                                                                    <i:if test="property.is(sirius.db.mixing.properties.EnumProperty.class)">
+                                                                        <i:local name="enumValue"
+                                                                                 value="@property.getValue(entity)"/>
+
+                                                                        @toUserString(enumValue)
+                                                                        <span class="border-sirius-gray tycho-tag d-inline-block">
+                                                                    @enumValue.as(java.lang.Enum.class).name()
+                                                                </span>
+
```

Comment:
```suggestion
                                                                      <i:local name="enumValue" value="@property.getValue(entity)"/>
                                                                      @toUserString(enumValue)
                                                                      <span class="pl-2 border-sirius-gray tycho-tag d-inline-block">
                                                                         @enumValue.as(java.lang.Enum.class).name()
                                                                      </span>

```

Reply:
This tries to fix the code layout, nothing else...

Reply:
Just a thought - instead of always using gray labels, we could use an array of colors and the ordinal() of the enum sonstant and index into this array - this would always render the first enum green, the second blue etc etc.

Maybe we have such an array already somewhere (??)  - could also be added in another PR...